src/components/candidateContent/candidateFeedbacksItem/CandidateFeedbacksItem.jsx

```diff
@@ -30,7 +30,7 @@ const CandidateFeedbacksItem = ({ user, candidateInfo, handleEditClick }) => {
     feedbacks.length ? feedback.finalEvaluation : 0,
   );
   const [editMode, setEditMode] = React.useState(false);
-
+  const loggedInUserInfo = useSelector((state) => state.userInfo.userInfo);
```

Comment:
do you use any other user info except roleType? why not select roleType directly?

Reply:
Didn't think about it to be honest. I'll try it in upcoming commits